src/main/sensors/esc_sensor.h

```diff
@@ -51,3 +51,5 @@ void startEscDataRead(uint8_t *frameBuffer, uint8_t frameLength);
 uint8_t getNumberEscBytesRead(void);
 
 uint8_t calculateCrc8(const uint8_t *Buf, const uint8_t BufLen);
+
+uint16_t calcRpmFromErpm(int16_t eRPM);
```

Comment:
Better to use `int` for such functions without exact size specified, all calculations are carried out in full ints anyway.